ppr-ui/src/views/newMhrRegistration/HomeLocation.vue

```diff
@@ -77,11 +80,13 @@ export default defineComponent({
     HomeLandOwnership
   },
   setup () {
+
+    const { setMhrRegistrationOwnLand, setMhrLocation } = useStore()
```

Comment:
This is a fix for MHR Submission, `setMhrLocation` was in the wrong place at the wrong time...
@cameron-eyds

Reply:
Perfect, good find!